sdk/cosmos/azure-cosmos/pom.xml

```diff
@@ -277,6 +290,7 @@ Licensed under the MIT License.
                 <include>io.micrometer:micrometer-core:[1.7.0]</include> <!-- {x-include-update;io.micrometer:micrometer-core;external_dependency} -->
                 <include>org.slf4j:slf4j-api:[1.7.30]</include> <!-- {x-include-update;org.slf4j:slf4j-api;external_dependency} -->
                 <include>org.hdrhistogram:HdrHistogram:[2.1.12]</include> <!-- {x-include-update;cosmos_org.hdrhistogram:HdrHistogram;external_dependency} -->
+                <include>io.opentelemetry:opentelemetry-exporter-logging:[1.0.0]</include> <!-- {x-include-update;io.opentelemetry:opentelemetry-exporter-logging;external_dependency} -->
```

Comment:
This is temporary change due to bug in event api #22115 . Will revert before merging the PR